eessi/testsuite/tests/apps/osu.py

```diff
@@ -50,13 +50,13 @@ class osu_pt_2_pt(osu_benchmark):
     device_type = parameter([DEVICE_TYPES[CPU], DEVICE_TYPES[GPU]])
     # unset num_tasks_per_node from the hpctestlib.
     num_tasks_per_node = None
-    # Note: device_buffers variable is inherited from the hpctestlib class and adds options to the launcher
-    # commands based on what device is set.
-    device_buffers = 'cpu'
 
     @run_after('init')
     def run_after_init(self):
         """hooks to run after init phase"""
+        # Note: device_buffers variable is inherited from the hpctestlib class and adds options to the launcher
+        # commands based on what device is set.
```

Comment:
can you expand the comment a bit so it's clear why we are doing this?
```suggestion
        # commands (before setup) if not equal to 'cpu'. We set it to 'cpu'  here initially, to fully control which options are set.
```